linode/manager
# refactor: [M3-7859]: Upgrade MSW to 2.2.3

## Description 📝
This update brings MSW 2.2.3 to Cloud Manager

It is a [major update with breaking changes](https://mswjs.io/docs/migrations/1.x-to-2.x/), hence the large amount of files touched.

## Changes  🔄
- Update MSW version
- Regenerate file in public dir (needed to bump `yargs-parser` for that purpose)
- Update all handlers in `serverHandlers.ts` (it was not fun)
- Update all tests

## Preview 📷
No visual change to be expected

## How to test 🧪

### Prerequisites
- Get yourself familiar with the new API and breaking changes: https://mswjs.io/docs/migrations/1.x-to-2.x/
- checkout PR
- run `yarn clean`
- run `yarn && yarn up`

### Verification steps
- turn on MSW and confirm no regression with mock data
- run test suite and confirm all pass

## As an Author I have considered 🤔

*Check all that apply*
